docs/internal.md

```diff
@@ -1,33 +1,39 @@
 # Internals of Azure SignalR Service
 
-> NOTE: This document describes the use of Azure SignalR Service with ASP.NET Core SignalR for now.
-> Documentation about ASP.NET SignalR support will be added shortly.
-
 - [Server Connections](#server-connections)
 - [Client Connections](#client-connections)
 - [Transport Data between Client and Server](#transport)
 
-Azure SignalR Service is built on top of ASP.NET Core SignalR framework.
-You can start from a local ASP.NET Core SignalR application, then seamlessly move to Azure SignalR Service with a few lines of code change.
+Azure SignalR Service is built on top of ASP.NET Core SignalR framework, and it supports running both ASP.NET Core SignalR and ASP.NET SignalR. **SignalR** is used in below section as a general term for both ASP.NET Core SignalR and ASP.NET SignalR. 
+
+> To support ASP.NET SignalR, Azure SignalR Service reimplement ASP.NET SignalR data protocol on top of ASP.NET Core framework.

```

Comment:
"reimplements ASP.NET SignalR's data protocol on top of the ASP.NET Core framework."